www/src/data/sidebars/doc-links.yaml

```
@@ -490,6 +490,8 @@
           link: /docs/migrating-from-v1-to-v2/
         - title: Migrating from v0 to v1
           link: /docs/migrating-from-v0-to-v1/
+        - title: Upgrade Gatsby for minor or patch releases
```

```suggestion
        - title: Upgrade for Minor or Patch Releases
```
Updating to match the page title recommendation above, and to use Title Case.